libp2p/Session.cpp

```diff
@@ -198,10 +207,10 @@ bool Session::checkPacket(bytesConstRef _msg)
 void Session::send(bytes&& _msg)
 {
     bytesConstRef msg(&_msg);
-    clog(VerbosityTrace, "net") << "Sending " << capabilityPacketTypeToString(_msg[0]) << " to "
-                                << m_info.id << "@" << m_socket->remoteEndpoint();
+    LOG(m_netLoggerDetail) << capabilityPacketTypeToString(_msg[0]) << " to";
```

Comment:
`send` is called from the public function `sealAndSend`, so I think it's better to leave global logger here